priv/repo/migrations/20170429120741_add_sessions_table.exs

```diff
@@ -0,0 +1,26 @@
+defmodule Hexpm.Repo.Migrations.AddSessionsTable do
+  use Ecto.Migration
+
+  def up do
+    create table(:sessions) do
+      add :token, :binary, null: false
+      add :data, :jsonb, null: false
+
+      timestamps()
+    end
+
+    create index(:sessions, ["(data->>'username')"])
```

Comment:
I think this design is cleaner all in all. Because we don't have to put application specific things in the plug session handling.